routes/index.js

```diff
@@ -1,9 +1,20 @@
 var express = require('express');
 var router = express.Router();
+var path = require('path');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('index', { title: 'Express' });
 });
 
+router.get('/badgein', function(req, res) {
+
+
+    res.sendFile(path.join(__dirname+'/HTML/UserSignIn.html'));
```

Comment:
I believe @MikeOlivas found a way how to do relative paths, take a look here: 
https://github.com/LUCCA-Capstone/LUCCA/pull/3/files#diff-bb503f3d9047bcd4a5730559f1e16560R11

Reply:
Right. As I understood it, path.resolve([...paths]) will normalize relative paths into an absolute path for you. [https://nodejs.org/api/path.html#path_path_resolve_paths](https://nodejs.org/api/path.html#path_path_resolve_paths)